src/targets/gpu/fuse_ck.cpp

```diff
@@ -161,10 +203,69 @@ struct find_ck_gemm
     }
 };
 
+static auto is_mul_module(module& m)
+{
+    auto is_mul =
+        match::arg(0)(match::name("mul")(match::all_of[match::inputs()](match::name("@param"))));
+    return match_instruction(m, std::prev(m.end()), is_mul).result != m.end();
+}
+
+MIGRAPHX_PRED_MATCHER(is_pointwise_scale, instruction_ref ins)
+{
+    if(ins->name() != "pointwise")
+        return false;
+    if(ins->module_inputs().size() != 1)
+        return false;
+    return is_mul_module(*ins->module_inputs().front());
+}
+
+struct find_ck_gemm_softmax_gemm
+{
+    auto matcher() const
+    {
+        auto gemm1 =
+            match::skip(match::name("contiguous"))(match::name("dot")(is_ck_gemm().bind("gemm1")));
+        auto mul     = match::name("pointwise")(match::either_arg(0, 1)(
```

Comment:
can this pointwise module take more than two params ? Or should it be exactly two ? 
Reason i am asking is that `mul` matcher can match if there are more than two inputs to pointwise module. 

```
(match::all_of[match::inputs()](match::name("@param")))
```

Reply:
It should only take two parameters.

Reply:
okay. Better to enforce that using `nargs(2)`